src/nodes/delta/approximations/extended.jl

```diff
@@ -0,0 +1,15 @@
+using Parameters
+
+export DeltaExtended, ET
+
+@with_kw struct DeltaExtended{T}
```

Comment:
We need to reflect on the naming here. I understand the choice for `Extended` here, but `Extended Transform (ET)` is not really trivial in the literature. `Extended` originates its naming from the extended Kalman filter, which has this non-linearity in it that is approximated through linearization. However, in `ReactiveMP` we are not only limited to Kalman filters. In this context we are talking about an approximation method here, `Linearization` would therefore be a better naming.

Reply:
I agree; I would actually go for Linearization, as you did for the Invertible NN node. Technically both INN and Delta nodes should share the same approximations.